pycbc/workflow/jobsetup.py

```diff
@@ -1615,3 +1615,123 @@ def create_node(self, input_files, tags=None):
 
         return node, out_file
 
+class PycbcCreateInjectionsExecutable(Executable):
+    """ The class responsible for creating jobs
+    for ``pycbc_create_injections``.
+    """
+
+    current_retention_level = Executable.ALL_TRIGGERS
+    def __init__(self, cp, exe_name, ifo=None, out_dir=None,
+                 universe=None, tags=None):
+        super(PycbcCreateInjectionsExecutable, self).__init__(
+                               cp, exe_name, universe, ifo, out_dir, tags)
+
+    def create_node(self, config_file=None, seed=None, tags=None):
+        """ Set up a CondorDagmanNode class to run ``pycbc_create_injections``.
+
+        Parameters
+        ----------
+        config_file : File
+            A ``File`` for inference configuration file to be used
```

Comment:
``File`` here should probably be ``pycbc.workflow.core.File`` like in `PycbcInferenceExecutable`? This is one of the places where Travis is failing.

Reply:
This is a ``pycbc.workflow.core.File`` but I can make the doctoring more verbose.